src/algorithms/tonal/key.cpp

```diff
@@ -77,7 +77,9 @@ void Key::configure() {
       _useMajMin = false;
     }
   }
-  const char* keyNames[] = { "A", "Bb", "B", "C", "C#", "D", "Eb", "E", "F", "F#", "G", "Ab" };
+  // TODO: choose which set of key names we want to use
+  //const char* keyNames[] = { "A", "Bb", "B", "C", "C#", "D", "Eb", "E", "F", "F#", "G", "Ab" };
+  const char* keyNames[] = { "A", "A#", "B", "C", "C#", "D", "D#", "E", "F", "F#", "G", "G#" };
```

Comment:
We should be careful about this change as it would break compatibility with previous versions of Essentia. We previously used the key names proposed in this change but we switched from them in https://github.com/MTG/essentia/pull/814. Let's discuss this again. @palonso @angelfaraldo 

Related: https://github.com/MTG/essentia/pull/815

Maybe this can be resolved by having a parameter that defines a key name style to use.

Reply:
Addressed in 3089d2d8332ad24eae5bdd59e4bae1b1de134971